uktrade/data-workspace-frontend
# feat/DT-2059 Add Table: Upload CSV

This is the upload CSV page for the service of adding a table to an existing catalogue page. This work includes validation:

- Given a user hasn’t selected a CSV to upload and they press the continue button then an error message should appear above the text input and an error summary should appear at the top of the page saying “Select a CSV”
- Given a user has selected a CSV that contains special characters in the name and they press the continue button then an error message should appear above the text input and an error summary should appear at the top of the page saying “File name cannot contain special characters apart from underscores and hyphens”

**How to Test**
1. Navigate to /datasets
2. Add /add-table to the end of the dataset ID
3. Click through the flow to get to 'Upload CSV'
4. Try uploading no file then try uploading a .csv file with special characters, both should display the error messages above.
